common/datatest-stable/src/runner.rs

```diff
@@ -22,19 +22,91 @@ struct TestOpts {
     filter: Option<String>,
     #[structopt(long = "exact")]
     /// Exactly match filters rather than by substring
-    filter_exact: bool,
+    exact: bool,
     #[structopt(long, default_value = "32", env = "RUST_TEST_THREADS")]
     /// Number of threads used for running tests in parallel
     test_threads: NonZeroUsize,
     #[structopt(short = "q", long)]
     /// Output minimal information
     quiet: bool,
     #[structopt(long)]
-    /// We already can't capture anything but we don't want arg parsing to fail so this is a noop
+    /// NO-OP: don't capture stdout/stderr of each task, allow printing directly
     nocapture: bool,
     #[structopt(long)]
     /// List all tests
     list: bool,
+    #[structopt(long)]
+    /// NO-OP: Run only ignored tests
+    ignored: bool,
+    #[structopt(long)]
+    /// NO-OP: Run ignored and not ignored tests
+    include_ignored: bool,
+    #[structopt(long)]
+    /// NO-OP: Forces tests to run in-process when panic=abort
+    force_run_in_process: bool,
+    #[structopt(long)]
+    /// NO-OP: Excludes tests marked as should_panic
+    exclude_should_panic: bool,
+    #[structopt(long)]
+    /// NO-OP: Run tests and not benchmarks
+    test: bool,
+    #[structopt(long)]
+    /// NO-OP: Run benchmarks instead of tests
+    bench: bool,
+    #[structopt(long)]
+    /// NO-OP: Write logs to the specified file instead of stdout
+    logfile: Option<String>,
+    #[structopt(long)]
+    /// NO-OP: Skip tests whose names contain FILTER (this flag can be used multiple times)
+    skip: Option<Vec<String>>,
```

Comment:
I know this is a noop, but you should be able to drop the option around the Vec

Reply:
It's required to pass multiple --skip=A --skip=B as the doc line indicates that the user can,

Otherwise you'd receive this error:

Running target/debug/deps/artifacts-e3e2958fabda3186
error: The argument '--skip <skip>' was provided more than once, but cannot be used multiple times

Reply:
You need to tweak the structopt attribute:
`#[structopt(long, number_of_values = 1)]`

The number_of_values attr will make this work with the type being just a `Vec`

Reply:
thanks, clear.

Reply:
eg: https://github.com/libra/libra/blob/master/x/src/test.rs#L20-L22

Reply:
@bmwill - done.   Yeah, no idea how this can be inferred from reading the docs.  Even using an Option<Vec<String>> was more of a guess than a declared feature for multi-use args. https://docs.rs/structopt/0.2.15/structopt/clap/struct.Arg.html#method.number_of_values.
Seems their could be some edge case around  --skip, but it may be that their is more expressivity/degrees of freedom/ in the struct mapping than exist in command line itself, making an empty Vec otherwise unachievable via command line inputs?   At least that's what this usage of number_of_values seems to imply.

Reply:
The issue is that structopt is a wrapper around clap so some things are less discoverable, not take clap's documentation is super clear. But fwiw what you had before `Option<Vec<String>>` without setting the `number_of_values` param would actually result in only the last instance of the option being respected (older instances would be overwritten).